app/posts/apply-for-teacher-training/2021-07-23-end-of-cycle-2021.md

```diff
@@ -0,0 +1,226 @@
+
+## Timeframe to carry over an application
+
+We have not set an end point for carrying over an application.
+
+This means that the [carry over message when a candidate's application does not lead to a place](/apply-for-teacher-training/end-of-cycle-2021/#if-it%E2%80%99s-too-late-to-apply-in-the-same-recruitment-cycle%2C-they-need-to-%E2%80%98carry-over%E2%80%99-their-application) and the [unsubmitted carry over message](/apply-for-teacher-training/end-of-cycle-2021/#they-need-to-%E2%80%98carry-over%E2%80%99-their-application-to-continue-working-on-it) continue to show indefinitely until the candidate actions it.
+
+Wherever we refer to applying for courses starting in a particular year, such as in carry over messages or banners, it should always reflect the year the candidate is eligible to start a course in.
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "carry-over-year-later.png"
+  },
+  caption: "Example: if a candidate started an application when it was possible to get on a course starting in the 2021 to 2022 academic year, but does not submit in time and does not sign back in until July 2023."
+}) }}  
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-no-longer-possible-apply-again-year-later.png"
+  },
+  caption: "Example: if a candidate submitted an application when it was possible to get on a course starting in the 2021 to 2022 academic year, but does get a place in time and does not sign back in until July 2023."
+}) }}  
+
+We're assessing whether we should set an end point.
+
+## Letting candidates know that courses are open
+
+Anyone who started or submitted an application for courses starting in the 2021 to 2022 academic year without securing a place gets an email letting them know when courses are open.
+
+```html
+<!-- if candidate name provided -->
+Dear [candidate name]
+<!-- endif -->
+
+<!-- if they started an application in the previous cycle but did not submit by their deadline -->
+# Applications are open - submit your teacher training application
+
+Complete your application to get on a course starting in the [year range] academic year:
+
+[link to sign in]
+<!-- endif -->
+
+<!-- if they submitted an application in the previous cycle but did not get a place in time  -->
+# Applications are open - apply for teacher training again
+
+You could get on a course starting in the [year range] academic year.
+
+Make any changes to your previous application and apply again:
+
+[link to sign in]
+<!-- endif -->
+
+# Get help to complete your application
+
+Talk to a teacher training adviser to get help writing a strong application:
+
+https://adviser-getintoteaching.education.gov.uk/
+
+```
```

Comment:
I wonder if we can use the Notify email preview style for this logic? (See [this example](https://bat-design-history.netlify.app/apply-for-teacher-training/end-of-cycle-and-carry-over/#for-candidates-who-get-a-rejection-after-18-september)… but maybe that makes this more complicated and difficult to understand?)

Reply:
Or… maybe we show 3 separate emails, with the logic in the copy instead?

Reply:
Good idea. I'll come back to this another time. Just want to get it recorded officially for now.